### DISCUSSION TITLE
Is PKCS fully supported?

### DISCUSSION THREAD

**Discussion by sawft99** [matteocorti/check_ssl_cert#499] [Q&A] [answered]
Based on some of the issues submitted before it looks like there has been work put into supporting PKCS. However, the Nagios description specifically mentions only supporting x509.

**Comment by matteocorti** [comment]
Should work. I'll update the documentation

**Comment by matteocorti** [comment]
The mention of x509 is pretty old: where did you find it?

**Reply by sawft99** [reply]
https://exchange.nagios.org/directory/Plugins/Network-Protocols/HTTP/check_ssl_cert/details

**Comment by matteocorti** [comment, marked as answer]
Thanks I updated both pages
